notebooks/azureml_model_headers_demo/german_credit_azure_ml_demo.ipynb

```diff
@@ -45,7 +45,7 @@
     "- scikit-learn=0.20.3\n",
     "- numpy=1.16.2\n",
     "- pandas\n",
-    "- azureml-sdk=1.4.0\n",
+    "- azureml-sdk=1.18.0\n",
```

Comment:
shouldn't this change go to `certifai_azure_model_env.yml` as well?

Reply:
The myenv.yaml is used inside the azure base container that already has this dependency of azure-ml. Run it once and to verify it works for you as well by checking updating this dependency